drivers/pwm/pwm_stm32.c

```diff
@@ -227,6 +254,7 @@ static int pwm_stm32_pin_set(const struct device *dev, uint32_t pwm,
 	const struct pwm_stm32_config *cfg = dev->config;
 
 	uint32_t channel;
+	uint32_t is_channel; /* complementary output if used */
```

Comment:
`is_channel` is not quite meaningful.
What about `channel_n_channel` ? (or anything better)

Reply:
proposal for `current_channel`